Fix/align channel with graph (#150)
NordicSemiconductor/pc-nrfconnect-dtm: Changelog.md

## 2.3.1 - UNRELEASED

## Fixed

> -   Side panel settings for channel selection did not match the graph responses

Reviewer: ```suggestion
-   Channel selection on the side panel did not match the chart.
```
